Makefile

```
@@ -306,11 +306,17 @@ install-docs: doc
 	$(INSTALL) -m 755 contrib/*.sh $(DESTDIR)$(docdir)/contrib || true
 	$(INSTALL) -m 755 contrib/*.rb $(DESTDIR)$(docdir)/contrib || true
 	$(INSTALL) -m 755 contrib/*.pl $(DESTDIR)$(docdir)/contrib || true
+	$(INSTALL) -m 755 contrib/*.py $(DESTDIR)$(docdir)/contrib || true
 	$(MKDIR) $(DESTDIR)$(docdir)/contrib/colorschemes
 	$(INSTALL) -m 644 contrib/colorschemes/* $(DESTDIR)$(docdir)/contrib/colorschemes || true
 	$(MKDIR) $(DESTDIR)$(docdir)/contrib/getpocket.com
 	$(INSTALL) -m 755 contrib/getpocket.com/*.sh $(DESTDIR)$(docdir)/contrib/getpocket.com || true
 	$(INSTALL) -m 644 contrib/getpocket.com/*.md $(DESTDIR)$(docdir)/contrib/getpocket.com || true
+	$(MKDIR) $(DESTDIR)$(docdir)/contrib/image-preview
+	$(INSTALL) -m 755 contrib/getpocket.com/vifmimg $(DESTDIR)$(docdir)/contrib/image-preview || true
+	$(INSTALL) -m 755 contrib/getpocket.com/nbrun $(DESTDIR)$(docdir)/contrib/image-preview || true
+	$(INSTALL) -m 755 contrib/getpocket.com/nbparser $(DESTDIR)$(docdir)/contrib/image-preview || true
+	$(INSTALL) -m 644 contrib/getpocket.com/README.org $(DESTDIR)$(docdir)/contrib/image-preview || true
```

I believe the source directory is incorrect (`getpocket.com` should be `image-preview`):
```suggestion
	$(INSTALL) -m 755 contrib/image-preview/vifmimg $(DESTDIR)$(docdir)/contrib/image-preview || true
	$(INSTALL) -m 755 contrib/image-preview/nbrun $(DESTDIR)$(docdir)/contrib/image-preview || true
	$(INSTALL) -m 755 contrib/image-preview/nbparser $(DESTDIR)$(docdir)/contrib/image-preview || true
	$(INSTALL) -m 644 contrib/image-preview/README.org $(DESTDIR)$(docdir)/contrib/image-preview || true
```

---

Indeed! Thanks. This also got me curious why we have all those `|| true` in there; if not for them, CI would've complained about the non-existent source paths. I added a commit that removes them (see its message for details if you're curious).